Common/src/Pollster/TaskHandler.cs

```diff
@@ -1059,69 +1079,40 @@ private async Task HandleErrorInternalAsync(Exception         e,
     {
       messageHandler_.Status = QueueMessageStatus.Processed;
     }
+    else if (cancellationToken.IsCancellationRequested && e is OperationCanceledException)
```

Comment:
With the change to `HandleErrorInternalAsync`, `requeueIfUnavailable` is not used anymore. You can refactor this out.